packages/myst-to-react/src/code.tsx

```diff
@@ -80,12 +82,16 @@ export function CodeBlock(props: Props) {
   );
 }
 
-const code: NodeRenderer<Code> = (node) => {
+const code: NodeRenderer<Code & { executable: boolean }> = (node) => {
```

Comment:
Unsure if this type extension is something to push back into myst-spec

Reply:
This does exist in `myst-spec-ext` at the moment, and could be imported from there.